can/uwrt_mars_rover_can_2022.yaml

```diff
@@ -7,8 +7,694 @@ uwrt_mars_rover_can_bus:
   nodes:
   - GIMBAL
   - JETSON
+  - ODRIVE
   - PDB
   - SCIENCE
   roboteq_canids:
     0: TEMP
-  messages: null
+  messages:
+  - ODRIVE_heartbeatMessage_axis0:
+      comment: Heartbeat messages for odrive
+      id: 1
+      size: 8
+      senders:
+      - ODRIVE
+      receivers:
+      - JETSON
+      frequency: 0
```

Comment:
Heartbeat messages are sent at 100ms, so the frequency here should be 10 (for 10Hz)